app/scripts/modules/core/src/pipeline/config/stages/executionWindows/executionWindows.html

```diff
@@ -2,7 +2,7 @@
   <div class="col-md-9 col-md-offset-1">
     <div class="checkbox">
       <label>
-        <input type="checkbox" ng-false-value="undefined" ng-model="stage.restrictExecutionDuringTimeWindow"/>
+        <input type="checkbox" ng-false-value="undefined" ng-checked="stage.restrictExecutionDuringTimeWindow" />
```

Comment:
Makes sense to me. Probably worth removing the `ng-false-value` attribute, since it's no longer being applied to a model.